src/components/Menu/MenuItem.tsx

```diff
@@ -93,7 +93,7 @@ class MenuItem extends UIComponent<any, any> {
         {childrenExist(children) ? (
           children
         ) : (
-          <a
```

Comment:
Do we really want to replace the <a> tag with a button? I think that might have its own set of issues. I don't understand the root issue though. Is it that the onclick handler is interfering with the default link?